src/main/java/fi/metatavu/metamind/server/rest/translation/KnotTranslator.java

```diff
@@ -33,6 +35,8 @@ public Knot translateKnot(fi.metatavu.metamind.persistence.models.Knot jpaKnot)
     result.setName(jpaKnot.getName());
     result.setStoryId(jpaKnot.getStory() != null ? jpaKnot.getStory().getId() : null);
     result.setType(jpaKnot.getType());
+    result.setCoordinates(coordinates.x(jpaKnot.getCoordinateX()));
```

Comment:
this is bit confusing, set coordinate x and y and then set coordinates to result only once (coordinates.setX(x), coordinates.setY(y) .... result.setCoordinates(coordinates))